src/shared/proto/tbots_robot_status_2014.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto2";
+
+message Tbots_RobotStatus2014 {
```

Comment:
This is probably getting very close to the 100 byte radio limit if the goal is to eventually send this over radio from the old robots. 

We probably need to do some pruning to get it just under 100 bytes + the extra stuff in `TbotsRobotMsg`

```
     // The link quality of the last received packet, from 0 (worst) to 1 (best).
     required double link_quality = 18;

     // The received signal strength of the last received packet, in decibels.
     required int32 received_signal_strength_db = 19;

     // Whether or nott the build ID information is valid.
     required bool build_ids_valid = 20;

     // The microcontroller firmware build ID.
     required uint32 fw_build_id = 21;

     // The FPGA bitstream build ID.
     required uint32 fpga_build_id = 22;
```

We probably don't need these @garethellis0 are we currently using this stuff?

Reply:
Not to the best of my knowledge, but I defer to the team historian @MathewMacDougall

Reply:
The only field I've seen used in any way is the `fw_build_id`, where we would get warnings if the ID mismatched with running software. Everything else I've never seen used (in any real capacity).

Unless we plan on supporting a similar build ID system going forwards, I'd say remove it. It's not useful in the legacy stack, and if won't use it in the future no point having it.

Reply:
The build ID system is a very good idea IMO, especially if we ever move to sharing dynamic params between the computer and the robot, we definitely want to know if we have a mismatch.